src/extension/features/general/budget-quick-switch/components/budget-list-item.jsx

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { getRouter } from 'toolkit/extension/utils/ember';
+// import { getRouter } from 'toolkit/extension/utils/ember';
```

Comment:
in the future, just delete code instead of commenting it out (i'll take care of this on this change)